ui/config/environment.js

```diff
@@ -23,7 +23,7 @@ module.exports = function(environment) {
     APP: {
       blockingQueries: true,
       mirageScenario: 'smallCluster',
-      mirageWithNamespaces: false,
+      mirageWithNamespaces: true,
```

Comment:
Just wondering what this does?

Reply:
It creates fixtures for [Nomad namespaces](https://www.nomadproject.io/guides/governance-and-policy/namespaces.html) which manifest as a dropdown in the left-hand gutter menu.